README.md

```diff
@@ -1,2 +1,4 @@
 # Pullreq
 Pull request repository
+
+echo "welcome"
```

Comment:
It was good
can you make echo "welcome to github"

Reply:
Its done.